pyproject.toml

```diff
@@ -1,63 +1,78 @@
 [build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
-
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
 
 [project]
 name = "draccus"
-version = "0.6"
+version = "0.1"
```

Comment:
revert. it's already on pypi

Reply:
Ooh ok - did not know it was already on pypi; I got rid of the publish to PyPI Github action because switching to setuptools vs. hatch; should I add back?

Reply:
I imagine we'll publish by hand anyway? but I don't really care